x-pack/filebeat/module/suricata/eve/ingest/pipeline.json

```diff
@@ -49,11 +54,44 @@
     }
   , {"convert":
       {"field": "suricata.eve.http.hostname"
-      ,"target_field": "url.hostname"
+      ,"target_field": "url.domain"
+      ,"type": "string"
+      ,"ignore_missing": true
+      }
+    }
+  , {"convert":
+      {"field": "suricata.eve.http.url"
+      ,"target_field": "url.path"
+      ,"type": "string"
+      ,"ignore_missing": true
+      }
+    }
+  , { "script":
+      { "lang": "painless"
+      , "source": "String[] split(String s, String sep) { def p=s.indexOf(sep); return p>=0? new String[]{s.substring(0,p), s.substring(p+1)} : new String[]{s};} def u=ctx['url']; def p=u!=null? u['path'] : null; if(p!=null) {def q=split(p,'#');if(q.length>1)u['fragment']=q[1];q=split(q[0],'?');if(q.length>1)u['query']=q[1];u['path']=q[0];}"
+      }
+    }
+  , {"convert":
+      {"field": "suricata.eve.http.hostname"
+      ,"target_field": "destination.domain"
       ,"type": "string"
       ,"ignore_missing": true
       }
     }
+  , {"convert":
+      {"field": "suricata.eve.http.http_refer"
```

Comment:
You have a typo in the Suricata field name, `http_refer` is missing an `er` :-)

Reply:
In my tests and in the docs it shows up as `http_refer`
https://redmine.openinfosecfoundation.org/projects/suricata/wiki/EveJSONFormat